Eatery Blue/UI/Profile/StayLoggedInSheet.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  StayLoggedInSheet.swift
+//  Eatery Blue
+//
+//  Created by Jayson Hahn on 11/7/23.
+//
+
+import Foundation
+import UIKit
+
+class StayLoggedInSheet: SheetViewController {
+    
+    private let descriptionLabel = UILabel()
+    private var delegate : AttemptLogin?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setUpStackView()
+    }
+    
+    let desc = "Due to updated Cornell security policies users are automatically logged out of their account after 24 hours."
+    let instruction = "Log back in to see your current total BRBs, Meal Swipes, City Bucks and Laundry."
```

Comment:
This should be factored out to a Constants file. I can't remember if Eatery has one, but if not, just mark this as `private`.